.gitignore

```diff
@@ -4,6 +4,7 @@
 *.iws
 *.jar
 *.sw?
+*.md
```

Comment:
What markdown files are we ignoring? Most of the time if these are docs they should be pushed with the content.